pkg/container/container_test.go

```diff
@@ -15,6 +22,83 @@ var _ = t.Describe("Container", func() {
 		}
 		sboxConfig = &pb.PodSandboxConfig{}
 	})
+	t.Describe("AddMountToSpec", func() {
+		It("should add the mount to the spec", func() {
+			sut.AddMountToSpec("test", "test", "test", []string{"test"})
+			Expect(len(sut.GetSpec().Mounts())).To(Equal(7))
```

Comment:
can we have a const value for the number of default mounts (6 is it?) then calculate what we should Equal() in relation to that? `defaultNumMount + mountsAdded`

Reply:
fixed!